sensor/DeviceServiceScanSensor.js

```diff
@@ -55,56 +61,129 @@ class DeviceServiceScanSensor extends Sensor {
     setInterval(() => {
       this.checkAndRunOnce();
     }, interval);
+
+    extensionManager.registerExtension(policyKeyName, this, {
+      applyPolicy: this.applyPolicy
+    });
   }
 
   isSensorEnable() {
-    return fc.isFeatureOn("device_service_scan");
+    return fc.isFeatureOn(featureName);
+  }
+  async applyPolicy(host, ip, policy) {
+    log.info("Applying device service scan policy:", ip, policy);
+    try {
+      if (ip === '0.0.0.0') {
+        this.scanSettings[ip] = policy;
+      } else {
+        if (!host)
+          return;
+        switch (host.constructor.name) {
+          case "Tag": {
+            const tagUid = host.o && host.o.uid;
+            if (tagUid) {
```

Comment:
we have a similar function to do this `HostManager.getTagMacs()`
implement something like `HostManager.getTagHosts()` instead of duplicate code